### ISSUE TITLE
provide a `Part.to_dict()` method that return the part properties as a dictionary

### ISSUE THREAD

**Issue by jberends** [KE-works/pykechain#131]
eg

```
>>> bike = project.part('Bike')
>>> print(bike.to_dict())
{ 'rear_wheel': True,
  'front_wheel': False,
  'frame': 'women'}
```

**Comment by jberends** [comment]
will be released in 1.9; see PR #133

**Comment by jberends** [comment]
@nawijn -- we made this method on your request.
